app/components/App.tsx

```diff
@@ -59,7 +58,7 @@ const getLocationBrowser = () => new Promise((resolve, reject) => {
     console.log(msg);
     reject(msg);
   }
-}, { timeout: 10000 });
```

Comment:
Never seen this before, but is not in the promises API, so if it was ever supported it isn't now.

Reply:
Weird, maybe it was a bad refactor or merge conflict resolution in the past. Definitely glad that TypeScript is helping us catch these kinds of things!